experimental/graphql.sample/README.md

```diff
@@ -0,0 +1,256 @@
+# GraphQL Testing
+
+## Getting started
+1. Get docker and docker-compose working
+2. `docker-compose up -d`
```

Comment:
This failed for me, because I already had a container using `:80`. An instruction to ensure nothing in Docker or local is using `:80` beforehand would be good.